### DISCUSSION TITLE
[질문] MainNavigatorImpl    SelectCharacterActivity 접근 안됨

### DISCUSSION THREAD

**Discussion by yoonhyeokCho** [TEAM-DAME/Dame-Android#97] [2. Q&A] [answered]
### 어떤 문제가 있을까요 ?

![image](https://user-images.githubusercontent.com/90389363/203067456-ea78d8fa-988e-4a86-946f-a4794f2a07a7.png)
![image](https://user-images.githubusercontent.com/90389363/203067582-dc938371-0cae-430c-93cd-bbd9e5850aaa.png)
![image](https://user-images.githubusercontent.com/90389363/203068177-79827df9-2c2b-4658-8e57-ab2df62e34f2.png)

**Comment by ChaMinXD** [comment, marked as answer]
새로 pull 해왔으면 gradle sync 가 안됐을수도 ? local property 에 X_NAVER 관련 2줄 추가하면 gradle sync 가 되면서 불러와졌어요 discord 방 참고
